.gitignore

```diff
@@ -20,3 +20,5 @@ contrib/build/*/tmp/*
 .pkg
 .kube
 .var
+build-image
+push-build-image
```

Comment:
can we put a dot in front of these to avoid cluttering our dir with empty files?
We can still have nice build targets if we want/need:
```
.PHONY: build-image

build-image: .build-image
    ... 
```